lib/dashboard/index.js

```diff
@@ -96,18 +76,33 @@ app.get('/panel/:bundle/*', ncgUtils.authCheck, (req, res, next) => {
 	}
 });
 
-function cacheDashboard() {
+function cacheDashboard(bundles) {
 	cachedDashboard = jade.renderFile(path.join(__dirname, 'src/dashboard.jade'), {
-		bundles: bundles.all(),
+		bundles,
 		publicConfig: configHelper.filteredConfig,
 		privateConfig: configHelper.config
 	});
 }
 
-// When all bundles are first loaded, render and cache the dashboard
-bundles.on('allLoaded', cacheDashboard);
+function setupBowerComponents(bundles) {
```

Comment:
This is a change you've made previously and that I had undone, because it broke the ability to add new panels during runtime. I must have removed my comment explaining this: https://github.com/nodecg/nodecg/blob/7edef60aaa321ee57c3a1d79b36bf59bb57b94ad/lib/dashboard/index.js#L48-L51